src/test/scala/com/cognite/spark/datasource/TimeSeriesRelationTest.scala

```diff
@@ -64,6 +64,7 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
           .sql(s"""
                 |select '$updatedDescription' as description,
                 |name,
+                |name as legacyName,
```

Comment:
Remove this from tests, we don't want to change the external schema and it doesn't look like we are. :+1:

Reply:
Done! Had make a new case class to get correct schema, but no big deal I guess.